SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+The .NET Core and ASP.NET Core support policy, including supported versions can be found at the [.NET Core Support Policy Page](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
```

Comment:
Is this relevant here?

Reply:
No. Also removed with the readme update.